src/web/helpers/api/flux-id.ts

```diff
@@ -61,7 +61,7 @@ const updateUser = (info?, shouldDispatchFromOauthTab = false) => {
         info,
       };
       fluxIDEvents.emit('update-user', currentUser);
-      if (shouldDispatchFromOauthTab) {
+      if (isWebSocialSignIn) {
         window.opener.dispatchEvent(new CustomEvent('update-user', {
```

Comment:
Use `if (isWebSocialSignIn && window.FLUX.version === 'web') {` or `window.opener?.dispatchEvent` because 
`window.opener` is `null` is null in electron version